evm-contracts/bsc-accounting-contract/src/AccountingContract.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract AccountingContract is Ownable {
+    constructor(
+        address initialOwner,
+        address initialWorker
+    ) Ownable(initialOwner) {
+        require(initialWorker != address(0), "Worker can't be zero address");
+        require(initialOwner != address(0), "Owner can't be zero address");
+        admin = initialOwner;
+        worker = initialWorker;
+    }
+
+    address public admin;
+    address public worker;
+    mapping(address => uint256) public nonces;
+
+    struct Payout {
+        uint256 amount;
+        bool paid;
```

Comment:
Is there any case where `paid` is `false`?

Reply:
There is no such case, i was thinking something else when writing this struct 😅 , i've removed it.